test/export/test_export.py

```diff
@@ -756,10 +756,10 @@ def test_buffer_util(self):
                 num_buffer += 1
                 buffer.append(get_buffer(ep, node))
         self.assertEqual(num_buffer, 3)
+        buffer_shape_set = set([x.shape for x in buffer])
+        expected = {torch.Size([]), torch.Size([100]), torch.Size([100])}
 
-        self.assertEqual(buffer[0].shape, torch.Size([100]))  # running_mean
-        self.assertEqual(buffer[1].shape, torch.Size([100]))  # running_var
-        self.assertEqual(buffer[2].shape, torch.Size([]))  # num_batches_tracked
+        self.assertEqual(buffer_shape_set, expected)
```

Comment:
This is because of ```for node in ep.graph.nodes``` doesn't have a preserved order, so we should do set comparison. Checked with the author (@cccclai) of this unit test, who is fine with this change.

Reply:
hmm, I guess it was non-deterministic before, this PR just changed the order due to the compile?

Reply:
I think so.